backstage/values.yaml

```diff
@@ -81,6 +81,18 @@ externalSecrets:
       key: AUTH_GOOGLE_CLIENT_ID
     - name: google-client-secret
       key: AUTH_GOOGLE_CLIENT_SECRET
+    - name: circleci_auth_token
+      key: CIRCLECI_AUTH_TOKEN
+    - name: auth_okta_client_id
```

Comment:
I think the okta ones should be removed?

Reply:
thanks, I removed them